doc/release-notes-28113.md

```diff
@@ -0,0 +1,6 @@
+RPC Wallet
+----------
+
+- The `signrawtransactionwithkey` and `descriptorprocesspsbt` calls now return
```

Comment:
nit: `signrawtransactionwithwallet` and `walletprocesspsbt` also affected

Reply:
Ugh, how did I miss those, will re-push.